doc/sflow/sflow_hld.md

```diff
@@ -68,9 +68,8 @@ sFlow will be implemented in multiple phases:
 2. sFlow extended router support.
 

```

Comment:
@rajkumar38 Do you know whether all 3 above phases are completed? if yes, mentioning the current status would be helpful.

Reply:
> @rajkumar38 Do you know whether all 3 above phases are completed? if yes, mentioning the current status would be helpful.

To our understanding, there is no progress on phase II and III. 
It is not clear if anyone else is already started working on this.